aws.go

```diff
@@ -114,12 +143,14 @@ func (a *AWS) Upload(local, remote string) error {
 			Key:         aws.String(remote),
 			Body:        file,
 			ContentType: aws.String(contentType),
+			ContentEncoding: aws.String(contentEncoding),
```

Comment:
Difficult, set by extension leads often to wrong enconding...
Let it empty is ugly...
Any ideas?